### PR TITLE
Fastsim premix wfs

### PR DESCRIPTION
**Pull request by lveldere**
replaces #8692 
rebased to CMSSW_7_5_0_pre3

### CODE REVIEW

**Comment by lveldere** [comment]
please test

**Comment by lveldere** [comment]
please test

the failure in the previous run of tests seems like a temporary proxy problem...

File: Configuration/PyReleaseValidation/python/relval_premix.py [hunk 1]
```diff
@@ -36,3 +36,12 @@
...
 workflows[500200.1]=['ProdZEE_13_pmx50ns',['ProdZEE_13','DIGIPRMXUP15_PROD_PU50','RECOPRMXUP15PROD_PU50']]
+
+# fastsim premix at 13 TeV and POSTLS1
+workflows[250399]=['',['FS_PREMIXUP15_PU25']]
+
+
+# fastsim 25ns pile up overlay using premix
+# TO BE UNCOMMENTED IN NEXT RELEASE
+#workflows[250402]=['',['FS_TTbar_13_PRMXUP15_PU25','HARVESTUP15FS','MINIAODMCUP15FS']]
+#workflows[250407]=['',['FS_NuGun_UP15_PRMXUP15_PU25','HARVESTUP15FS','MINIAODMCUP15FS']]
```
Comment (lveldere) [inline Configuration/PyReleaseValidation/python/relval_premix.py]: i comment out all workflows that depend on the premixed minbias sample

**Comment by lveldere** [comment]
@franzoni 

i had to open a new pr, because I am not allowed top reopen #8692 

your request:
"""
1) you comment out all workflow's dependent on the existence of the premix sample
2) you include in relval_premix.py only the workflows which will have a regular fast.vs.full validation comparison (not to load the IB process beyond what we need)
3) can you please remove the print statement at line 823
4) to avoid temporary failures in the IB, we suggest you comment out the lines 813-824
Cheers, Phat and Giovanni
"""

1) done: see comments on code
2) done: see comments on code
3) done
4) all code that could lead to crashes in IB tests is removed by reacting on your requests 1) and 2),
so I don't understand what I need to do with lines 813-824. Probably I don't understand your request.

**Comment by lveldere** [comment]
@srimanob, @nclopezo, @boudoul, @franzoni 

may I remind you of this pr?
I understood PdmV and FastSim agreed on these changes.

**Comment by lveldere** [comment]
please test

**Comment by franzoni** [comment]
for the record: 
this PR provides the ingredients to kick-start the premixing@fastSim in production
it cannot be integrated as is: the premixed sample needs to be made first (in the making).
Phat and Giovanni

**Comment by lveldere** [comment]
outdated

**Pull request closed without merging**